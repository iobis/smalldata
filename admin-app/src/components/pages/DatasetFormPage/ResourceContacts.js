import ContactsForm from './ContactsForm/ContactsForm'
import React from 'react'
import { useTranslation } from 'react-i18next'

export default function ResourceContacts(props) {
  const { t } = useTranslation()

  return (
    <ContactsForm
      className="resource-contacts"
      contactsTableHeader={t('datasetFormPage.resourceContacts.contactsTableHeader')}
      {...props}/>
  )
}
