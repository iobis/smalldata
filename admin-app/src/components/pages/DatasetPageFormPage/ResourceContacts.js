import ContactsForm from './ContactsForm/ContactsForm'
import React from 'react'

export default function ResourceContacts(props) {
  return <ContactsForm className="resource-contacts" {...props}/>
}
