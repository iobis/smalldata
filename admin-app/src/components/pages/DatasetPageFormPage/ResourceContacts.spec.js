import ResourceContacts from './ResourceContacts'
import React from 'react'
import { mount } from 'enzyme'

describe('ResourceContacts', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <ResourceContacts
        data={[]}
        onChange={jest.fn()}/>)
    expect(wrapper).toMatchSnapshot()
  })
})
