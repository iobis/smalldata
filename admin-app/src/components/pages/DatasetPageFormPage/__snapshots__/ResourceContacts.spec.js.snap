// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourceContacts renders correctly 1`] = `
<ResourceContacts
  data={Array []}
  onChange={[MockFunction]}
>
  <ContactsForm
    className="resource-contacts"
    data={Array []}
    onChange={[MockFunction]}
  >
    <div
      className="contacts-form resource-contacts"
    >
      <InputText
        className="name"
        name="datasetPageFormPage.contactsForm.name"
        onChange={[Function]}
        value=""
      >
        <div
          className="column field name"
        >
          <label
            className="label"
          >
            datasetPageFormPage.contactsForm.name.label
          </label>
          <input
            className="input"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </InputText>
      <InputText
        className="email"
        name="datasetPageFormPage.contactsForm.email"
        onChange={[Function]}
        value=""
      >
        <div
          className="column field email"
        >
          <label
            className="label"
          >
            datasetPageFormPage.contactsForm.email.label
          </label>
          <input
            className="input"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </InputText>
      <InputText
        className="organisation"
        name="datasetPageFormPage.contactsForm.organisation"
        onChange={[Function]}
        value=""
      >
        <div
          className="column field organisation"
        >
          <label
            className="label"
          >
            datasetPageFormPage.contactsForm.organisation.label
          </label>
          <input
            className="input"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </InputText>
      <InputText
        className="position"
        name="datasetPageFormPage.contactsForm.position"
        onChange={[Function]}
        value=""
      >
        <div
          className="column field position"
        >
          <label
            className="label"
          >
            datasetPageFormPage.contactsForm.position.label
          </label>
          <input
            className="input"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </InputText>
      <div
        className="column field"
      >
        <button
          className="add button"
          onClick={[Function]}
        >
          common.add
        </button>
      </div>
      <div
        className="column field"
      >
        <table
          className="general table is-fullwidth is-striped is-hoverable"
        >
          <thead>
            <tr>
              <th>
                datasetPageFormPage.contactsForm.name.label
              </th>
              <th>
                datasetPageFormPage.contactsForm.email.label
              </th>
              <th>
                datasetPageFormPage.contactsForm.organisation.label
              </th>
              <th>
                datasetPageFormPage.contactsForm.position.label
              </th>
            </tr>
          </thead>
          <tbody />
        </table>
      </div>
    </div>
  </ContactsForm>
</ResourceContacts>
`;
